app/helpers/navigation_items_helper.rb

```diff
@@ -40,17 +40,17 @@ def secondary_items(path, current_organisation_id)
         NavigationItem.new("Schemes", "/organisations/#{current_organisation_id}/schemes", subnav_supported_housing_schemes_path?(path)),
         NavigationItem.new("Users", "/organisations/#{current_organisation_id}/users", subnav_users_path?(path)),
         NavigationItem.new("About this organisation", "/organisations/#{current_organisation_id}", subnav_details_path?(path)),
-        (NavigationItem.new("Housing providers", housing_providers_organisation_path(current_user.organisation), housing_providers_path?(path)) if FeatureToggle.managing_owning_enabled?),
-        (NavigationItem.new("Managing agents", managing_agents_organisation_path(current_user.organisation), managing_agents_path?(path)) if FeatureToggle.managing_owning_enabled?),
+        (NavigationItem.new("Housing providers", "/organisations/#{current_organisation_id}/housing-providers", housing_providers_path?(path)) if FeatureToggle.managing_owning_enabled?),
```

Comment:
why not using the path helpers?

Reply:
For support users the path should include the selected org id not the id of the org the user belongs to. Otherwise they can't access other org's pages. Thinking about it again `housing_providers_organisation_path(current_organisation_id)` should have the same effect